Moblin/Various/Model.swift

```diff
@@ -5076,32 +5130,32 @@ extension Model {
         )
     }
 
-    private func sendSpeedAndTotalToWatch() {
+    private func sendSpeedAndTotalToWatch(speedAndTotal: Any) {
```

Comment:
Use `Any` as little as possible for lower risk of mistakes.

Reply:
done